packages/design-system/src/styles/components/_Table.scss

```diff
@@ -132,18 +137,12 @@ $table-dense-padding: $spacer-half $spacer-2 !default;
     }
   }
 
-  // stylelint-disable selector-no-qualifying-type
-  th[data-title],
-  td[data-title] {
-    // Add column heading via psuedo element
-    &::before {
-      content: attr(data-title);
-      display: block;
-      font-weight: bold;
-      padding-bottom: $spacer-half;
-    }
+  // Add column heading for stacked table
+  .ds-c-table--stacked__col-header {
+    display: block;
+    font-weight: bold;
+    padding-bottom: $spacer-half;
```

Comment:
@davidakennedy - the column header text instead of pseudo element fixes the issue of repeated information.

Reply:
Same feedback [here](https://github.com/CMSgov/design-system/pull/927/files#r570382699). Lets hold off on addressing that issue in this ticket, we should file a new ticket for that and properly assess the different options before picking a solution

Reply:
Ok, have reverted back to using pseudo element. I've changed the <tr> display to `table-row`. Would that help address the `Firefox/NVDA` navigation issue?

Reply:
@davidakennedy = Here is an example of [responsive stacked table on hc.gov](https://www.healthcare.gov/third-party-privacy-policies/). The thead is `display: none` when stacked whereas we've just hidden the header display but still made it visible to screen reader. Do you think we should just make it `display: none`?